### DISCUSSION TITLE
Quick Toggle betweenmodes

### DISCUSSION THREAD

**Discussion by erankatz2020** [tefkah/zotero-night#103] [Ideas]
Hi. I love this plugin, my eyes thank you!

I wanted to ask if there's a way to add a quick toggle button on the UI, or a shortcut to move between dark and light mode. thanks.

**Comment by tefkah** [comment]
Glad you like it!
There should be a quick toggle on the main menu, do you not see it?
![image](https://user-images.githubusercontent.com/21983833/223085617-e83a51a0-0caf-4f5e-809d-f526c947524d.png)

**Reply by tefkah** [reply]
(im referring to the sun emoji)

**Comment by erankatz2020** [comment]
oh!  yes. I didn't see in a PDF tab but in the main window I do.
If it was possible to assign a hotkey that would also be great.

thanks!
